EnglishFilter/sections/specific.txt

```diff
@@ -304,6 +304,7 @@ $third-party,script,domain=readcomiconline.li|filmyzilla.autos|gaybeeg.info|haca
 !
 !
 gg.deals###top-banner-image
+apk4all.com##.box > p.field > a.button
```

Comment:
Actually, we can do as @Yuki2718 suggested like `apk4all.com##.control.field > a.is-danger.button` or `apk4all.com##.block + p.field > a.is-danger`